CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# dbt_tiktok_ads v0.8.1
+
```

Comment:
Since we're not including a Schema Changes matrix, should we note "This release does not introduce any schema changes" or something like that at the top?